flutter/beagle_components/lib/beagle_image.dart

```diff
@@ -64,36 +63,47 @@ class _BeagleImageState extends State<BeagleImage> {
 
   void downloadImage() {
     final RemoteImagePath path = widget.path;
-    widget.imageDownloader.downloadImage(path.url).then((value) {
-      setState(() {
-        imageBytes = value;
+    try {
+      widget.imageDownloader.downloadImage(path.url).then((value) {
+        setState(() {
+          imageBytes = value;
+        });
       });
-    });
+    } on Exception catch (e, _) {
+      developer.log(e.toString());
```

Comment:
What do you think of always using a tag  like `BEAGLE_TAG = "BeagleSDK"` to identify beagle logs just like Beagle Android does?
```suggestion
      developer.log(
          e.toString(),
          name: BEAGLE_TAG,
    );
```

Reply:
There will be a specific PR for logging. You can be sure that your comment will be taken into account